Create tac-talk-guidelines.md (#293)
confidential-computing/governance: tac-talk-guidelines.md

Due to this mix, where the only real common ground is an interest in confidential computing, technical talks can be very interesting (and slightly unpredictable) to both the presenter and the audience. When presenting, you need to keep in mind that what you see as obvious in your work might not be known to all members in the audience. That being said, we do appreciate any talk and often enough, the Q&A after the talk ends in interesting debates.

### TL;DR
- No product pitches, just technical discussions on public topics
- 20-30 minutes + following Q&A
- Meetings are recorded and must follow Linux Foundation antitrust policies
- [LFX meeting calendar](https://confidentialcomputing.io/get-involved/calendar/) for meeting invite
- An email link to suggest a talk is at the bottom

> ## Meeting Format

Reviewer: Maybe condense this with the TL;DR section.. not sure there's much different esp. if the calendar / times are linked per the next comment.

Author: Good point, I added the tl;dr at the end when I saw that it was a large document. But I now took the `no advertisements` point into the next list and removed the tl;dr completely.

> - Technical presentations are typically 20-30 minutes
> - Followed by Q&A session (duration varies based on audience engagement)
> - Total presentation slot usually does not exceed 45 minutes
> - All meetings are public and open for anyone to participate

Author: ```suggestion
- Technical presentations are typically 20-30 minutes
- Followed by Q&A session (duration varies based on audience engagement)
- Total presentation slot usually does not exceed 45 minutes
- All meetings are public and open for anyone to participate
- No product pitches, just technical discussions on public topics
```

- The recording of the meeting will later be published on [youtube](https://www.youtube.com/@confidentialcomputingconso5871/videos).
- Meetings are held biweekly on Thursdays at 7am US Pacific Time (4pm Central European Time)
- Note: Time differences may vary during daylight savings transitions

> - You can find the meeting invite and time in your time zone [on the LFX meeting calendar](https://confidentialcomputing.io/get-involved/calendar/)

Reviewer: Consider referring to just the LFX calendar and/or the times in the governance repo. Should we ever shift the meeting time or calendar URL we don't want too many places to fix it.
https://github.com/confidential-computing/governance/tree/main/TAC/Meetings#meeting-details

Author: ```suggestion
- You can find the meeting invite and time in your time zone [on the LFX meeting calendar](https://confidentialcomputing.io/get-involved/calendar/). For more information on the TAC meetings refer to [meeting details](https://github.com/confidential-computing/governance/tree/main/TAC/Meetings#meeting-details)
```

## Talk Categories

> Our talks can losely be grouped into three categories:

Reviewer: typo
```suggestion
Our talks can loosely be grouped into three categories:
```

1. **Open Source Talks**
   - Project presentations. You did something and want to share it with the community.
   - Paper results. You published something with an open-source artifact and want to share it with the community.
   - Technical demos. You have a specific setup that is documented in an open-source repository and want to show it to the community so that it can be re-used.

2. **Information Dissemination Talks**
   - Educational content about approaches/technologies/methods. You think not enough people know about a topic but have no agenda to push one solution.
     - These can be more in a lecture-style
     - Treat this similar as an introductory lecture to a somewhat general audience

> 3. **Interest Group/Discussion**
>     Sometimes, we want to engage with other communities or similar-minded organizations. Thus, not all talks have to be technical. These could be:
>    - Focused discussions on specific topics or organizations
>    - Community engagement on particular technical areas

Reviewer: typo -
```suggestion
3. **Interest Group/Discussion**
   - Sometimes, we want to engage with other communities or similar-minded organizations. Thus, not all talks have to be technical. These could be:
   - Focused discussions on specific topics or organizations
   - Community engagement on particular technical areas
```

- Technology choices and their rationale
- Threat models and trust models
- End-to-end system considerations
- System security architecture
- Technical challenges and solutions
- Implementation bottlenecks and their resolution

### Content Restrictions
Due to anti-trust regulations, technical talks must be limited to:
1. Open-source projects that are public
2. General technical discussions that extend beyond specific open-source projects

If you are uncertain, you can refer to the [Linux Foundation antitrust policy](https://www.linuxfoundation.org/legal/antitrust-policy) or ask one of the TAC members.

## Presentation style and best practices
- Focus on technical content rather than product pitches
- Avoid marketing-style presentations
- Sales pitches should be directed to the outreach committee
- Technical solutions and research results are welcome
- Focus on security architecture and implementation details
- Prepare for technical questions from a diverse audience
- Consider the varying technical backgrounds of attendees
- Avoid proprietary or patent-specific content
- Maintain focus on open-source and general technical discussions

## Contact and Resources

> If you want to give a talk or have an idea for an interesting topic, feel free to send an email to [tac@lists.confidentialcomputing.io](mailto:tac@lists.confidentialcomputing.io?subject=Talk%20suggestion%3A%20%3Cyour%20idea%3E&body=Hi%20everyone!%20I%20have%20an%20idea%20for%20a%20talk%3A%0D%0A%0D%0ATitle%20or%20topic%3A%20%3CSpecific%20title%20or%20topic%20idea%3E%0D%0ARelated%20URL%3A%20%3Crepository%2Fpaper%20link%3E%0D%0APotential%20presenters%3A%20%3Cyou%2Fsomeone%20else%2Fsomeone%20you%20are%20suggesting%3E%0D%0AStyle%3A%20%3Csee%20TAC%20talk%20guidelines%20about%20talk%20categories%3E)

Reviewer: One last change.. make it clear they are emailing a list and not a person.
```suggestion
If you want to give a talk or have an idea for an interesting topic, feel free to send an email to the public mail list [tac@lists.confidentialcomputing.io](mailto:tac@lists.confidentialcomputing.io?subject=Talk%20suggestion%3A%20%3Cyour%20idea%3E&body=Hi%20everyone!%20I%20have%20an%20idea%20for%20a%20talk%3A%0D%0A%0D%0ATitle%20or%20topic%3A%20%3CSpecific%20title%20or%20topic%20idea%3E%0D%0ARelated%20URL%3A%20%3Crepository%2Fpaper%20link%3E%0D%0APotential%20presenters%3A%20%3Cyou%2Fsomeone%20else%2Fsomeone%20you%20are%20suggesting%3E%0D%0AStyle%3A%20%3Csee%20TAC%20talk%20guidelines%20about%20talk%20categories%3E)
```
